packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,67 @@
       <protected>false</protected>
       <shortDescription>Opens in the browser tab</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_LookAtDeveloperConsole</fullName>
+      <value>Look at the developer console for more information.</value>
```

Comment:
@MehdiMitiche Guessing that the developer console is on another screen, I would only change the verb at the beginning for: 
"Go to the developer console for more information."